src/style.css

```diff
@@ -0,0 +1,28 @@
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  font-size: 0;
+}
+
+.container {
+  display: flex;
+  flex-flow: column wrap;
```

Comment:
sorry, missed this one.
there is no property `column wrap` for flex-flow. the reason it works is that it ignores the `wrap` part 😆 
so `flex-flow: column` is the way to go.

another way of putting blocks with stars in rows would be setting `flex-direction: column`
JFYI